client/src/components/Donation/DonationModal.js

```diff
@@ -88,8 +92,8 @@ function DonateModal({
 
   const donationText = (
     <b>
-      Become a supporter and help us create even more learning resources for
-      you.
+      Become a $5 / month supporter and help us create even more learning
```

Comment:
I do not like the idea of static amounts in code. Quincy always gets us to keep tweaking this, please pull this from the config.

Reply:
I will tweak the config and the UI in an upcoming pr. if that is alright with you, it would be better to handle it there.